python/lib/sift_py/ingestion/config/telemetry.py

```diff
@@ -220,17 +221,26 @@ def _from_yaml(
                         }
                     )
 
+            if rule.get("asset_names") or rule.get("tag_names"):
```

Comment:
Ideally we would do this warning in `validate_rules()` but since that's looking at the list of instantiated `RuleConfig`s and `[asset|tag]_names` are not a field, we can't really check for it.